src/Parser/DisciplinePriest/Modules/Features/AtonementDamageSource.js

```diff
@@ -0,0 +1,22 @@
+import Module from 'Parser/Core/Module';
+
+class AtonementDamageSource extends Module {
```

Comment:
This won't be 100% reliable since there's a delay between Atonement transfer, so if you did damage with Penance and 1ms later a PtW damage tick happens, that PtW will be the last damage event before the Atonement heals go through. It's close enough though I think.